androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -78,6 +84,12 @@ fun EnhancedVideoPlayer(
 
     val isFullScreen = orientation == Configuration.ORIENTATION_LANDSCAPE
     var isSettingsOpen by rememberSaveable { mutableStateOf(false) }
+    var selectedTrack by remember {
+        mutableStateOf(TrackQualityItem(builder = TrackSelectionParameters.Builder()))
```

Comment:
This `Builder()` is deprecated. Try to use a non deprecated method

Reply:
Done